package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "bluebird": "2.2.2",
     "bunyan": "1.0.0",
-    "fxa-content-server-l10n": "git://github.com/mozilla/fxa-content-server-l10n.git#4168ef3cb7",
+    "fxa-content-server-l10n": "git://github.com/mozilla/fxa-content-server-l10n.git",
```

Comment:
Hrm. We can keep this locked down but it needs to be updated to the latest hash before extracting strings. This is so the extracted strings are merged with the latest template from the l10n repo.

@dannycoates What's the significance of the hash this is currently locked to? Is there a policy for updating it?